src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -96,7 +102,7 @@ const SidebarSidecarLayoutContent = ({
 					{sidebarContent}
 				</div>
 			</MobileMenuContainer>
-			<div className={s.contentWrapper}>
+			<div className={s.contentWrapper} ref={contentRef}>
```

Comment:
We now track scroll progress through `contentWrapper`, rather than through the whole page.